src/ZPublisher/HTTPRequest.py

```diff
@@ -1644,18 +1644,19 @@ class TemporaryFileWrapper(_TemporaryFileWrapper):
     and doesn't complain about such a move either.
     """
 
-    unlink = staticmethod(unlink)
-    isfile = staticmethod(isfile)
+    if PY2:
+        unlink = staticmethod(unlink)
+        isfile = staticmethod(isfile)
```

Comment:
I think they were made into staticmethods, to avoid becoming bound methods and magically getting `self` as an additional call argument. Since these are simple functions, that'd fail.